src/config.js

```diff
@@ -108,6 +108,17 @@ let loadUserConfig = function (configDir: string, configOptions: ConfigOptions)
   return config
 }
 
+function shell (onWindows: boolean = false): string {
+  let shellPath
+  if (onWindows) {
+    shellPath = (process.env['COMSPEC'] && process.env['COMSPEC'].split(/\\|\//)) || (process.env['SHELL'] && process.env['SHELL'].split(`/`))
```

Comment:
@RasPhilCo this is flipped I think, from what I can tell is that both `$SHELL` and `$COMSPEC` are defined in `Git Bash` and `$COMSPEC` is set to `cmd.exe` because Windows?